fe/fe-core/src/main/java/org/apache/doris/nereids/memo/Group.java

```diff
@@ -139,7 +165,28 @@ public List<GroupExpression> clearPhysicalExpressions() {
     }
 
     public double getCostLowerBound() {
-        return costLowerBound;
+        return -1D;
```

Comment:
I think -0.01 is not an acceptable bound. who about return 0.

Reply:
this is original value, just remove useless variables